modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -20,7 +20,7 @@ This allows more related data to be fetched by fewer read operations, resulting
 The older store formats are based on the original `standard` format, each with different features and limitations.
```

Comment:
We don't need to talk about the benefits of standard and aligned. Just say they are deprecated and link to the detailed section on their deprecation.